ios/FluentUI/Core/Theme/Tokens/GlobalTokens.swift

```diff
@@ -6,9 +6,49 @@
 import CoreGraphics
 
 /// Global Tokens represent a unified set of constants to be used by Fluent UI.
-struct GlobalTokens {
-    struct Colors {
-        enum Neutral: TokenSet {
+public struct GlobalTokens {
```

Comment:
Note that `GlobalTokens` are now public. This will be necessary in the future when these tokens come from an external module, and allows clients to access them directly as well.

Discussion point: what is the role of `MSFColors` in a world where colors can easily be grabbed from a `GlobalTokens` or `AliasTokens` struct?

Reply:
not to mention all the objc clients grabbing colors... I think last time design team was horrified to find out grayXXX was directly grabbed in client code (which we do) and we should push for client team to grab the alias or control token as much as possible

Reply:
Good point. I'm going to hide the tokens for now and expose these colors through a new API, and we can think more about how to expose these properly in the future.